packages/client/src/comp/WarningMsg.test.tsx

```diff
@@ -1,20 +1,53 @@
 import React from 'react'
-import { render, fireEvent, act } from '@testing-library/react'
+import { render, fireEvent, act, wait } from '@testing-library/react'
 import { WarningMsg } from './WarningMsg'
 import { UnstatedContainer } from "./StateContainer"
+import { MemoryRouter } from 'react-router-dom'
+import { Initialize } from './Initialize'
+import { mocked } from 'ts-jest/utils'
+import { client } from '../lib/trpc'
+jest.mock('../lib/trpc.ts')
+jest.mock('../lib/analytics.ts')
+
+const renderResult = async () => {
+  const utmFields = {
+    utm_source: 'source1234',      //eslint-disable-line @typescript-eslint/camelcase
+    utm_medium: 'medium1234',      //eslint-disable-line @typescript-eslint/camelcase
+    utm_campaign: 'campaign1234',  //eslint-disable-line @typescript-eslint/camelcase
+    utm_term: 'term1234',          //eslint-disable-line @typescript-eslint/camelcase
+    utm_content: 'content1234',    //eslint-disable-line @typescript-eslint/camelcase
+  }
+
+  const params = Object.entries(utmFields).map(([key, val]) => `${key}=${val}`).join('&')
+
+  const initialData = mocked(client, true).fetchInitialData = jest.fn().mockResolvedValue({
+    type: 'data',
+    data: { emailFaxOfficials: false, org: {} },
+  })
+
+  const rendered = render(
+    // Using MemoryRouter is recommended way to test react-router-dom
+    // https://reacttraining.com/react-router/web/guides/testing
+    <MemoryRouter initialEntries={['/org/default?' + params]}>
+      <Initialize/>
+      <WarningMsg/>
+    </MemoryRouter>,
+    { wrapper: UnstatedContainer },
```

Comment:
Could we have just passed in `<InitialDataContainer>` with `emailFaxOfficials: false` set?